src/extensions/x-error-group.yaml

```diff
@@ -0,0 +1,29 @@
+openrpcExtension: "0.0.0-development"
```

Comment:
There's now an official specification to pull from in the repo https://github.com/open-rpc/tools/blob/main/packages/extensions/src/x-error-groups/x-error-groups.json. it has a oneOf at the top to explicitly signal that ref's are supported, and better external docs etc...